lens-common/info.rkt

```diff
@@ -3,12 +3,11 @@
 (define collection 'multi)
 
 (define deps
-  '("base"
+  '(("racket" "6.3")
```

Comment:
Nit: specifying the version without the `#:version` keyword is [deprecated](http://docs.racket-lang.org/pkg/metadata.html)